lua/sql/defs.lua

```diff
@@ -529,6 +607,80 @@ M.type_of_stmt_ptr = ffi.typeof "sqlite3_stmt*"
 M.type_of_exec_ptr = ffi.typeof "int (*)(void*,int,char**,char**)"
 M.type_of_blob_ptr = ffi.typeof "sqlite3_blob*"
 
+---Fail at lua side if sql table doesn't exists.
+---@param db table
+---@param tbl_name string
+---@param method any
+M.assert_tbl = function(db, tbl_name, method)
+  local errmsg = ("sql.nvim: can not execute %s, %s doesn't exists."):format(method, tbl_name)
+  assert(db:exists(tbl_name), errmsg)
+end
```

Comment:
this is out of place. This file is about ffi definitions

Reply:
👍